externals/CMakeLists.txt

```diff
@@ -175,4 +204,22 @@ add_wn_externals(
     marl
 )
 
+add_wn_externals(
+  rlottie
+    DIRECTORY
+    rlottie
+  COMPILE_OPTIONS
+    ${WN_RLOTTIE_ADDITIONAL_OPTIONS}
+)
+
+add_wn_externals(
+  freetype
+    DIRECTORY
+    freetype
+  COMPILE_OPTIONS
+    ${WN_ADDITIONAL_FREETYPE_OPTIONS}
+)
+
+include(RmlUI.txt)
```

Comment:
Did you just do this separately since this file is getting pretty large?

Reply:
Yeah, I figured we can start having separate *.txt for anything that we are overloading the build.